config/default.yml

```diff
@@ -75,7 +75,10 @@ AllCops:
   AllowSymlinksInCacheRootDirectory: false
   # What version of the Ruby interpreter is the inspected code intended to
   # run on? (If there is more than one, set this to the lowest version.)
-  TargetRubyVersion: 2.0
+  # If a value is specified for TargetRubyVersion then it is used.
+  # Else if .ruby-version exists and it contains an MRI version it is used.
+  # Otherwise the default (2.0) ruby is used.
```

Comment:
I don't like the final sentence - it should be something like `Otherwise we fallback to the oldest officially supported Ruby version (2.0)`.